core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt

```diff
@@ -764,9 +765,16 @@ abstract class CoreReaderFragment :
     }
     toolbar?.let(::setUpDrawerToggle)
     setDrawerLockMode(DrawerLayout.LOCK_MODE_UNLOCKED)
-    closeAllTabsButton?.setImageDrawable(
-      ContextCompat.getDrawable(requireActivity(), R.drawable.ic_close_black_24dp)
-    )
+    closeAllTabsButton?.apply {
+      setImageDrawable(
+        ContextCompat.getDrawable(requireActivity(), R.drawable.ic_close_black_24dp)
+      )
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
```

Comment:
The title is not showing when I am long clicking on the `close All tabs` button. You should set this in `showTabSwitcher()` method instead of `hideTabSwitcher()`.

Reply:
>The title is not showing when I am long clicking on the close All tabs button.
 
@Sagar0-0 Me too! Please be carefull. Please let us know if you don't understand any part of the code.

Reply:
Which API level you're testing on? I can see the title PFA SS, I tested on 33.

Reply:
For other changes can you see the titles?

Reply:
@Sagar0-0 I have tested it on three different devices. 
1. API level 32.
2. API level 34.
3. API level 31.

On these devices title is not showing for this. However, if you put this in `showTabSwitcher` then it will work because this is the right place where you should put this code.

> For other changes can you see the titles?

yes we can see the title for other changes.

> Which API level you're testing on? I can see the title PFA SS, I tested on 33.

I don't know how you can see the title, but you are putting the code in the wrong place. 
`hideTabSwitcher` is used when we close the tabs, and `showTabSwitcher` is used when we are showing the tabs.